server/src/grpc/meta_event_service/mod.rs

```diff
@@ -74,7 +74,10 @@ macro_rules! handle_request {
 
                 let res = handle
                     .await
-                    .map_err(|e| Box::new(e) as _)
+                    .map_err(|e| {
+                        error!("Fail to process request from meta, err:{}", e);
```

Comment:
I think we should add this log in L92, it has more condition check `Ok(Err(e)) | Err(e)`

Reply:
Yes, the outer is runtime's error, mainly what we wanted is the error in the inner result...